apps/test/unit/code-studio/initApp/loadAppTest.js

```diff
@@ -186,7 +186,10 @@ describe('loadApp.js', () => {
         expect(writtenLevelId).to.be.undefined;
         expect(name).to.equal('_share_image.png');
         expect(blob).to.have.property('type', 'image/png');
-        expect(blob).to.have.property('size', 523181);
+        const expectedPngSize = /PhantomJS/.test(window.navigator.userAgent)
+          ? 523181 // PhantomJS
+          : 38961; // ChromeHeadless
+        expect(blob).to.have.property('size', expectedPngSize);
```

Comment:
It looks like ChromeHeadless writes a much, much smaller PNG than Phantom does.  Unfortunately, this is a tough one to make work both ways without weakening this assertion somehow, so I'm going with a little browser detection.  I'm open to pushback here - if we think we don't really care about the size of the generated PNG, we could just remove this assertion, or check that the size is at least 1k.

Reply:
Have you been able to visually inspect the PNG generated by ChromeHeadless to confirm that it's actually correct?

Reply:
Great question.  I haven't for this case, but I'm going through a similar exercise for a different test that's comparing dataURIs, and while the PNG comes out correct, the difference in size is astounding.

| PhantomJS | ChromeHeadless |
| --- | --- |
| 523kB | 40kB |
| ![expected](https://user-images.githubusercontent.com/1615761/71299500-6656ce00-2342-11ea-9508-4fb7da0254dc.png) | ![expected-chrome](https://user-images.githubusercontent.com/1615761/71299503-6a82eb80-2342-11ea-9ecf-6f69a36555d9.png) |

Reply:
Wow. Go, ChromeHeadless!